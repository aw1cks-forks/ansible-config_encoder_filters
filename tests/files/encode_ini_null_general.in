var1: null
var2:
var3: '!!!null'
var4: some text
var5: 123
var6: 987.654
var7:
  - yes
  - "false"
