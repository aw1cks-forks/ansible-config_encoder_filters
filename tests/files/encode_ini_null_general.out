var3
var4=some text
var5=123
var6=987.654
var7=True
var7=false
